sdk/core/azure-core/tests/testserver_tests/test_rest_http_response.py

```diff
@@ -168,7 +168,7 @@ def test_json(send_request):
         request=HttpRequest("GET", "/basic/json"),
     )
     assert response.json() == {"greeting": "hello", "recipient": "world"}
-    assert response.encoding == 'utf-8-sig'  # for requests, we use utf-8-sig instead of utf-8 bc of requests behavior
```

Comment:
Do we still need these three tests? Maybe one is good enought?

Reply:
Not quite sure which tests you mean by the three tests, but I feel like more tests is better, so might as well not get rid of them